test/test_translator.py

```diff
@@ -84,27 +84,25 @@ def test_parse_incident() :
     ]
   }
 }
-
-
     assert test_feature == valid_feature
 
-    test_var=""" <incident id="U13631595_202012160845">
-        <creationtime>2020-12-16T08:45:03Z</creationtime>
-        <updatetime>2020-12-16T17:18:00Z</updatetime>
-        <type>CONSTRUCTION</type>
-        <description>Roadwork - Lane Closed, MERGE LEFT [Trafficade, iCone]</description>
-        <location>
-          <direction>ONE_DIRECTION</direction>
-          <polyline>34.8380671,-114.1450650,34.8380671,-114.1450650</polyline>
-        </location>
-        <starttime>2020-12-16T08:45:03Z</starttime>
-        </incident> """
+def test_parse_incident_no_data():
+  test_feature = icone_translator.parse_incident(None)
+  valid_feature=None
+  assert test_feature == valid_feature
 
-    icone_obj = xmltodict.parse(test_var)
-    test_feature = icone_translator.parse_incident(icone_obj['incident'])
+def test_parse_incident_invalid_data():
+  test_var = 'a,b,c,d'
+  callback = MagicMock()
+  test_feature = icone_translator.parse_incident(test_var, callback_function=callback)
+  assert callback.called
+
+def invalid_incident_callback(incident):
```

Comment:
Now that this is no longer used, can we remove?

Reply:
i removed it

Reply:
I still see this method in the code.

Reply:
@abinashkonersman Still seeing this dead function, can we remove?

Reply:
i removed the dead function